docs/modules/ROOT/pages/pallets/pallet_xcm.adoc

```diff
@@ -0,0 +1,340 @@
+:source-highlighter: highlight.js
+:highlightjs-languages: rust
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+
+= pallet_xcm
+
+Branch/Release: `release-polkadot-v{1.3.0}`
+
+== Purpose
+
+`pallet-xcm` is responsible for filtering, routing, and executing incoming XCM.
+
+== Glossary
+
+** `Barrier` -- whether to execute the XCM call at all
+** `Fungible Asset` -- any unit is worth the same
+** `NonFungible Asset` -- each unit is worth a different amount and/or is unique in some way
+** `Junction` -- single item in a path to describe a relative location (think of `dir` or `dir/$file_path` as a junction)
+** `Junctions` -- multiple `Junction`s, formed only through MultiLocation construction
+** `MultiAsset` -- Either an amount of a single fungible asset, or a single well-identified non-fungible asset.
```

Comment:
what does `well-identified` mean in this context? It might be confusing for some